ui/js/models/perf/alerts.js

```diff
@@ -75,25 +75,56 @@ treeherder.factory('PhAlerts', [
                 this.updateStatus(status);
             };
         });
-        AlertSummary.prototype.getTextualSummary = function() {
+        AlertSummary.prototype.getTextualSummary = function(copySummary = false) {
```

Comment:
Hmm, eslint doesn't seem to like this construction (see travis), I guess probably because it's ES6-only?

It's probably best just to take out the assignment from the prototype (it will still be falsey if you don't pass anything to the function, because it'll be undefined)